src/app/audiologist-login/audiologist-login.component.ts

```diff
@@ -14,11 +15,20 @@ import { ThsDataService } from '../services/ths-data.service';
 
 export class AudiologistLoginComponent {
   // Added audiologistUserName & changed (patientId --> audiologistID)
-  public audiologistUserName: string = ''; 
-  public audiologistID: string = '';
+  public audiologistUserName: string = '';
+  public audiologistPassword: string = '';
   public authenticationFlag: boolean = true;
 
-  constructor(private router: Router, private tsDataService: TsScreenerDataService, private tfiDataService: TfiDataService, private thsDataService: ThsDataService) {};
+  constructor(private router: Router, private tsDataService: TsScreenerDataService, private tfiDataService: TfiDataService, private thsDataService: ThsDataService, private serverAuthenticationService: ServerAuthenticationService) { }
```

Comment:
tsDataService, tfiDataService, and thsDataService are not being used here, or in the development branch.